TimerTriggerAttachmentsVerify/index.ts

```diff
@@ -9,9 +11,20 @@ const timerTrigger: AzureFunction = async (
   const timeStamp = new Date().toISOString();
   const client = df.getClient(context);
 
+  const config: Config = {
+    authTimeout: 3000,
+    host: getRequiredEnvVar("IMAP_HOST"),
+    password: getRequiredEnvVar("IMAP_PASSWORD"),
+    port: Number(getRequiredEnvVar("IMAP_PORT")),
+    tls: true,
+    user: getRequiredEnvVar("IMAP_MAIL")
+  };
+
+  // It resolves a problem at start time.

```

Comment:
it's still not clear. which kind of problem ? can we add a link to some resource ?

Reply:
this was tricky i did not understand why sometimes the TimerTrigger was not working,  I found this https://docs.microsoft.com/en-us/sandbox/functions-recipes/timer and solved the problem